config/routes.rb

```diff
@@ -1,5 +1,10 @@
 Rails.application.routes.draw do
   resources :books, only: %i[index show]
+  resources :rents, only: %i[create]
+
+  resources :users, only:[] do
+   resources :rents, only: %i[index]
```

Comment:
```suggestion
   resources :rents, only: %i[index create]
```

And add it in here, this way is it the same as apiary